From 221c063620a95be63d4e14c2933b1f81999d7452 Mon Sep 17 00:00:00 2001
From: Jordan Lazare <Jordan.Lazare@amd.com>
Date: Tue, 23 Feb 2016 12:14:16 -0500
Subject: [PATCH 0828/1110] drm/amd/dal: Remove wait_for_vblanks call on
 atomic_commit

It appears to be superfluous, and having the call was causing a performance regression

Signed-off-by: Jordan Lazare <Jordan.Lazare@amd.com>
Reviewed-by: Tony Cheng <tony.cheng@amd.com>
Acked-by: Harry Wentland <harry.wentland@amd.com>
---
 drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dm_types.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dm_types.c b/drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dm_types.c
index 84ce0bb..f469017 100644
--- a/drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dm_types.c
+++ b/drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dm_types.c
@@ -2391,8 +2391,6 @@ int amdgpu_dm_atomic_commit(
 				0);
 	}
 
-	drm_atomic_helper_wait_for_vblanks(dev, state);
-
 	/* In this state all old framebuffers would be unpinned */
 
 	drm_atomic_helper_cleanup_planes(dev, state);
-- 
2.7.4

